src/InvariantTest.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+import {Test} from "./Test.sol";
+
+contract InvariantTest is Test {
```

Comment:
I don't think we should inherit from `Test` here

It's common to have something like `ProtocolTest is Test` where `ProtocolTest` deploys contracts and scaffolds the system, and you'd want to be able to reuse that in your invariant tests.

If `InvariantTest` is standalone you can do `MyInvariantTests is ProtocolTest, InvariantTest` (with `MyInvariantTests is Test, InvariantTest` being the default way to setup invariant tests), but when `InvariantTest` inherits from `Test` you'd be inheriting from `Test` twice which is not allowed, so you can't easily reuse `ProtocolTest` with this